File: Java/백준/Bronze/28113. 정보섬의 대중교통/정보섬의 대중교통.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Main {
    public static void main(String[] args) throws IOException {

        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        String[] input = br.readLine().split(" ");
        int N = Integer.parseInt(input[0]); //지하철 승강장까지 걸어가는 데 걸리는 시간
        int A = Integer.parseInt(input[1]); //마지막 버스가 도착하는 시간
        int B = Integer.parseInt(input[2]); //마지막 열차가 도착하는 시간

        if (A < B) {
            System.out.print("Bus");
        } else if (A > B) {
            System.out.print("Subway");
        } else {
            System.out.print("Anything");
        }
    }
}

File: C11/백준/Bronze/28113. 정보섬의 대중교통/정보섬의 대중교통.c
#include <stdio.h>

int main(void){
	
	int N, A, B;
	scanf("%d %d %d", &N, &A, &B);
	
	if(A < B){
		printf("Bus");
	} else if(A > B){
		printf("Subway");
	} else{
		printf("Anything");
	}
	
	return 0;
}